src/components/note/MemberNoteAdminBlock.tsx

```diff
@@ -59,7 +59,7 @@ const MemberNoteCollectionBlock: React.FC<{ memberId: string; searchText: string
 
   const { insertMemberNote, updateLastMemberNoteAnswered, updateLastMemberNoteCalled } = useMutateMemberNote()
   const uploadAttachments = useUploadAttachments()
-  if (!currentMemberId || loadingNotes || errorNotes || !memberAdmin) {
```

Comment:
You can combine loading state and whether there are notes to achieve  that does not return Skelton element  when loading more.